app/views/volunteers/edit.html.slim

```diff
@@ -2,7 +2,12 @@
   .col-xs-12
     h1= t_title(:edit)
 
-== render 'form'
+.row
+  .col-xs-12
+    = simple_form_for(@volunteer) do |f|
```

Comment:
I see what you mean. But I think there is better ways to solve this. 

Just as a possible example amongst many, change `_content.scss` to: 
```scss
.row,
.simple_form {
  margin-bottom: $padding-large-horizontal;
}
```